### ISSUE TITLE
Make Beautiful

### ISSUE THREAD

**Issue by boxhead201** [mikekelly/hal-browser#75]
This UI is a good starting point for an administration GUI.  

For context, see http://stackoverflow.com/questions/34867014/javascript-crud-admin-for-spring-data-hateoas-backend

**Comment by mikekelly** [comment]
I agree. I don't have the time to work on this right now, but working on the UX/design of the browser is a high up on the priority list
